### ISSUE TITLE
searchable: false doesn't work for every column

### ISSUE THREAD

**Issue by denjs** [wenzhixin/bootstrap-table#2268]
Hi,
I've got these columns:

{
              field: 'manager',
              title: 'Manager',
              sortable: true,
              align: 'center',
              searchable: false
            },
            {
              field: 'physicalDeliveryOfficeName',
              title: 'Office',
              sortable: true,
              align: 'center',
              searchable: false
            },
            {
              field: 'l',
              title: 'Location',
              sortable: true,
              align: 'center',
              searchable: false
            }

they look the same, but only one with field 'physicalDeliveryOfficeName' actually excluded from search, other two can be search, what could possibly be an issue and where to look?
Thanks.

**Comment by dabros** [comment]
Please see https://github.com/wenzhixin/bootstrap-table/blob/master/CONTRIBUTING.md#bug-reports

A fiddle can help us prove this bug, then see a fix

**Comment by denjs** [comment]
That would be a really big one, I'll try to build up a fiddle, but when it is a simple case it works okay.

**Comment by dabros** [comment]
@denjs - you can take a subset of your data and use it as local array, feed that into a fiddle that mimics as much as possible real use case.

If server side, checkout http://doc.jsfiddle.net/use/echo.html

Id recommend start simple, just keep going till you can replicate the bug. Until its replicated, there is nothing we can do for you.

Also keep in mind obvious points such as whether the content you think is being found in column X is actually 'hidden' data (link href in anchor, or ect) in column Y.

**Comment by denjs** [comment]
Thanks for helping me, I'm trying to create a fiddle right now stuffing it up with more and more stuff from my code. It works as supposed to work... looks like it would be harder to catch that nasty bug as I thought ;)

**Comment by denjs** [comment]
ROFL, I found it... I don't know if it actually can be considered as a bug, probably not ;)
So, in my say first column I have a name of an employee and in my say second column I have a name of manager. So I disabled manager column as a searchable but in my name column I set up a popover with the name of manager as well ;) so search engine sees it as well ;) I don't know if we can search by only value of html elements but not in something like 'data-content=name1' for the bootstrap popover.
Stupid mistake, sorry about that.
